frontend/iframe/viewmodels/SessionViewModel.ts

```diff
@@ -87,7 +91,10 @@ export class SessionViewModel extends BaseSessionViewModel {
     _createRoomViewModelInstance(roomId) {
         const room = this.client.session.rooms.get(roomId);
         if (room) {
-            const roomVM = new RoomViewModel(super.childOptions({room}));
+            const roomVM = new RoomViewModel(super.childOptions({
+                room,
+                singleRoomMode: this._singleRoomId !== undefined
```

Comment:
Can `this._singleRoomId` ever be empty?

Reply:
When no room id or alias was set through the block inspector or the popup settings, or we failed to resolve the room alias, `this._singleRoomId` is `undefined`.

Reply:
ok cool then!